content/articles/ms-ignite-2018.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "OpenCensus at Microsoft Ignite"
+date: 2018-10-03
+draft: false
+weight: 3
+aliases: [/blog/ms-ignite-2018]
+hidden: true
+logo: /img/logo-sm.svg
+---
+
+## Open Census at Microsoft Ignite
+
+![](http://appanacdn.blob.core.windows.net/cdn/icons/aic.png)
+
+### Introduction
+Just three months ago [we announced that Microsoft joined the OpenCensus project](https://open.microsoft.com/2018/06/13/microsoft-joins-the-opencensus-project/), and last week at [Microsoft Ignite](https://www.microsoft.com/ignite) we were able to share a few more details about how OpenCensus will be leveraged and advanced.
+
+### OpenCensus spotlight
+
+During Ignite's keynote, Microsoft announced support for Azure Kubernetes Service (AKS) Virtual Nodes. Virtual Nodes allow applications to scale more quickly, in a matter of mere minutes. The demonstration included a highlight of Azure Monitor [Live
+Metrics Stream](https://docs.microsoft.com/azure/application-insights/app-insights-live-stream).
+
+[![Microsoft Ignite 2018 keynote AKS Virtual Nodes with Application
+Insights and Open Census](ms-ignite-2018-images/ms-ignite-kyenote.gif)](https://mediastream.microsoft.com/events/2018/1809/Ignite/player/tracks/track2.html?start=17300)
+
+The keen eyed among you may have noticed that this demo is powered by [OpenCensus](https://github.com/Azure-Samples/virtual-node-autoscale)!
+
+![OpenCensus in Ignite Keynote](ms-ignite-2018-images/keynote-demo-app-uses-open-census.png)
```

Comment:
Please move this image to `static/images/`, then update the path to `/images/keynote-demo-app-uses-open-census.png`

Reply:
Thank you for detailed instructions. Fixed. Is there a way I can check it?

I know other systems (like octopress) has the same issue - markdown-valid path for images wouldn't work when rendered. [Docfx](https://dotnet.github.io/docfx/) doesn't have this issue so I get used to it =)

Reply:
For a quick check, you can build the website locally using Hugo and navigate to the new page. The instructions are https://github.com/census-instrumentation/opencensus-website#contributing, then the updated opencensus.io will be served at localhost:1313.